automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala

```diff
@@ -30,48 +24,59 @@ import spray.json._
 import java.util.UUID
 import scala.concurrent.{Await, Future}
 import scala.language.postfixOps
+import org.broadinstitute.dsde.test.pipeline._
 
 @WorkspacesAzureTest
-class AzureWorkspacesSpec extends AnyFlatSpec with Matchers with CleanUp {
-  val owner: Credentials = UserPool.userConfig.Owners.getUserCredential("hermione")
-  val nonOwner: Credentials = UserPool.chooseStudent
-
-  private val azureManagedAppCoordinates = AzureManagedAppCoordinates(
-    UUID.fromString("fad90753-2022-4456-9b0a-c7e5b934e408"),
-    UUID.fromString("f557c728-871d-408c-a28b-eb6b2141a087"),
-    "staticTestingMrg",
-    Some(UUID.fromString("f41c1a97-179b-4a18-9615-5214d79ba600"))
-  )
+class AzureWorkspacesSpec extends AnyFlatSpec with Matchers with BeforeAndAfterAll with CleanUp {
+  // The values of the following vars are injected from the pipeline.
+  var billingProject: String = _
```

Comment:
How exactly does this work if you want to run the tests locally?

Reply:
You can set up local environment variables for the tests to use. Let me create some templates for that.